pytest_tests/pytest.ini

```diff
@@ -9,6 +9,9 @@ markers =
 #   special markers
     sanity: small tests subset
     staging: test to be excluded from run in verifier/pr-validation/sanity jobs and run test in staging job
+    nightly: test runs in nightly testrun
+    not_sbercloud: test not supported Sbercloud infrastructure
+    not_docker: development environment not supported
```

Comment:
I suppoose that "running in docker" is not equal to "running in neofs-dev-env" in general. Could you specify the marker description (or marker itself)?